scripts/vault-init.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ex
+
+sleep 10
+# shellcheck source=/dev/null
+source $1/.env
+export VAULT_ADDR=http://127.0.0.1:$VAULT_GUEST_PORT
+docker exec -i vault sh -c "vault operator init -key-shares=5 -key-threshold=3 --address=$VAULT_ADDR" > keys.txt
+while read line; do
```

Comment:
while running this, don't you have errors in the vault log?

Reply:
Didn't look for errors in the vault log

Reply:
missed the `head -n5 keys.txt` part
Should also work with `3` instead of `5` since that's what you need to unseal, but worth checking.
My bad :)

Reply:
I will replace this bash script later with python module than has a function for it.